include/swift/Basic/LangOptions.h

```diff
@@ -112,6 +112,9 @@ namespace swift {
     /// Detect and automatically import modules' cross-import overlays.
     bool EnableCrossImportOverlays = false;
 
+    /// Emit cross-import remarks.
```

Comment:
The doc comment here is a little tautological. Maybe something like "Emit a remark when import resolution implicitly adds a cross-import overlay"?